docker/windows.Dockerfile

```diff
@@ -8,6 +8,7 @@ WORKDIR /go/src/sigs.k8s.io/secrets-store-csi-driver
 ADD . .
 ARG TARGETARCH
 ARG TARGETOS
+ARG IMAGE_VERSION
```

Comment:
Do we use this ARG inside dockerfile?

Reply:
The ARG is passed to the builder and will get consumed as part of `make build`. If not set, the `IMAGE_VERSION` provided as build will not take effect instead the `IMAGE_VERSION` will Makefile will be used.